packages/protocol/script/deploy_on_l1.sh

```diff
@@ -9,6 +9,7 @@ ORACLE_PROVER=0xa0Ee7A142d267C1f36714E4a8F75612F20a79720 \
 SOLO_PROPOSER=0xa0Ee7A142d267C1f36714E4a8F75612F20a79720 \
 OWNER=0x70997970C51812dc3A010C7d01b50e0d17dc79C8 \
 TAIKO_L2_ADDRESS=0xa0Ee7A142d267C1f36714E4a8F75612F20a79720 \
+L2_SIGNAL_SERVICE=0x0000777700000000000000000000000000000007 \
```

Comment:
@davidtaikocha , how to figure out these values? I ran `pnpm test:genesis` and it fails.

Reply:
they are defined in https://github.com/taikoxyz/taiko-mono/blob/opimize_proving/packages/protocol/test2/genesis/test_config.json#L36, to make the L2 pre-deployed contract addresses deterministic

Reply:
`TAIKO_L2_ADDRESS= 0xa0Ee7A142d267C1f36714E4a8F75612F20a79720 ` is different from the one in test_config.json